packages/blocks/src/store/reducer.js

```diff
@@ -20,12 +20,21 @@ import { combineReducers } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
 /**
- * Module Constants
+ * @typedef {Object} WPBlockCategory
```

Comment:
It should probably be descriptive of the _block type_:

```suggestion
 * @typedef {Object} WPBlockTypeCategory
```

Reply:
> It should probably be descriptive of the _block type_:

Not going to change. See https://github.com/WordPress/gutenberg/pull/22192#discussion_r421788753.